mm2src/mm2_tests.rs

```diff
@@ -5504,6 +5504,258 @@ fn test_qrc20_tx_history() {
     }
 }
 
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_tx_history_segwit() {
+    let bob_passphrase = get_passphrase(&".env.seed", "BOB_PASSPHRASE").unwrap();
```

Comment:
`BOB_PASSPHRASE` is different across CI agents, so this test will likely fail on Win and Mac. Please use the hardcoded seed here. I've also pushed few CI hotfixes to dev, please sync your branch.